src/devices/Max31865/samples/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
```

Comment:
copyright header missing

Reply:
I've added the missing copyright header.